test/e2e/README.md

```diff
@@ -40,10 +40,16 @@ You can install all the dependencies with [brew](https://brew.sh/):
 If an action is required on your part, `brew` will let you know in its log
 messages.
 
+##### Fork and Clone Antrea
+* Fork https://github.com/vmware-tanzu/antrea on github
+* mkdir -p antrea/github.com/antrea; cd $_
+* git clone https://github.com/<your-name>/antrea
+
 #### Managing the cluster
 
 Use the following Bash scripts to manage the Kubernetes Nodes with Vagrant:
 
+* cd test/e2e
```

Comment:
this looks out-of-place here, given that this is meant to be specifically a list of available scripts, as per the sentence above